### PR TITLE
Add documentation for mobile: source endpoint

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Based on https://github.com/appium/appium-xcuitest-driver/pull/534

### CODE REVIEW

File: docs/en/writing-running-appium/ios-xctest-source.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Getting Page Source For iOS With WebDriverAgent/XCTest Backend
+
+WebDriverAgent provides a possibility to represent page source in different formats. Usually it is enough to call _getPageSource_ driver method, which is the standard one and is available for all the client libraries. This method returns the hierarchical representation of the current UI source represented as XML document. However, sometimes it is convenient to get page representation in JSON format or some other format, which is supported by WDA/XCTest. Also, the actual performance of different page representation providers is different and the default method is not always necessarily the most performant one.
```
Comment (imurchie) [inline docs/en/writing-running-appium/ios-xctest-source.md]: This makes it seem like "xml", "json" and some other format(s) are available, but the `format` description below says only the two are there.

**Review by imurchie — CHANGES_REQUESTED** [review]
I think that throughout what is in italics should be in code format (with backticks).

File: docs/en/writing-running-appium/ios-xctest-source.md (hunk 1 above)
Comment (mykola-mokhnach) [reply docs/en/writing-running-appium/ios-xctest-source.md]: yes, I want to add at least one more and there is a PR for that, however I don't know when it is going to be merged

**Review by jlipps — COMMENTED** [review]
I like this but this should be added as a new command doc, using the command template and doc generator that @dpgraham has made. We should in fact have a whole section for "mobile" commands, and this should be one of them. @dpgraham are you up for working on this branch with @mykola-mokhnach and helping to put this in that format?

**Comment by dpgraham** [comment]
Yeah, I can easily add a section to the commands documentation. I'll just cut-and-paste what's in this PR.

**Comment by mykola-mokhnach** [comment]
@imurchie What else should be done to have this document merged?

File: docs/en/writing-running-appium/ios-xctest-source.md (hunk 1 above)
Comment (imurchie) [inline docs/en/writing-running-appium/ios-xctest-source.md]: The `source` command has a page: `/docs/en/commands/session/source.md`

**Review by imurchie — COMMENTED** [review]
A couple of things:
1. I still would like all the things in _italics_ to be in `backticks`.
2. This should be in the ios folder (https://github.com/appium/appium/tree/master/docs/en/writing-running-appium/ios)
3. This should be linked to in the 'source' document (https://github.com/appium/appium/blob/master/commands-yml/commands/session/source.yml)

**Comment by mykola-mokhnach** [comment]
> I still would like all the things in italics to be in backticks.

Changed 

> This should be in the ios folder (https://github.com/appium/appium/tree/master/docs/en/writing-running-appium/ios)

Moved

> This should be linked to in the 'source' document (https://github.com/appium/appium/blob/master/commands-yml/commands/session/source.yml)

I'm not sure into which category this should be added. Any hints there?

**Comment by imurchie** [comment]
I would put it in the description,  in the source documentation.

**Comment by mykola-mokhnach** [comment]
NP, I can do that when the new documentation is published

**Comment by imurchie** [comment]
Just a link is all that is needed, I think. Something like "For iOS you can get a different underlying source structure by using [link]"

**Pull request closed without merging**